src/interceptors/ClientRequest/NodeClientRequest.ts

```diff
@@ -388,16 +371,16 @@ export class NodeClientRequest extends ClientRequest {
       this.requestBody = concatChunkToBuffer(chunk, this.requestBody)
     }
 
-    const resolvedRequestBody = bodyBufferToString(
-      Buffer.concat(this.requestBody)
-    )
-
+    const resolvedRequestBody = Buffer.concat(this.requestBody)
     this.log('resolved request body:', resolvedRequestBody)
 
-    return resolvedRequestBody
+    return resolvedRequestBody.buffer.slice(
```

Comment:
This looks like what `bufferUtils.encodeBuffer()` is doing, is it not? Can we use that utility here as well?